java/cuvs-java/src/main/java22/com/nvidia/cuvs/spi/JDKProvider.java

```diff
@@ -370,8 +478,10 @@ public void addVector(int[] vector) {
   }
 
   /**
-   * This {@link CuVSDeviceMatrix} builder implementation returns a {@link CuVSDeviceMatrix} backed by managed RMM
-   * device memory. It uses a {@link PinnedMemoryBuffer} to batch data before copying it to the GPU.
+   * This {@link CuVSDeviceMatrix} builder implementation returns a
+   * {@link CuVSDeviceMatrix} backed by managed RMM
+   * device memory. It uses a {@link PinnedMemoryBuffer} to batch data before
+   * copying it to the GPU.
```

Comment:
Sorry, why did this need reflowing?

Reply:
Sorry must have C++ auto-formatting accidentally applied.